app/scripts/models/user.js

```diff
@@ -360,8 +360,11 @@ define(function (require, exports, module) {
         .then(function () {
           var isSignUp =
             account.get('verificationReason') === VerificationReasons.SIGN_UP;
+          var emailSent = !! account.get('emailSent');
 
-          if (! account.get('verified') && isSignUp) {
+          // Only send account verification email, it signing up
+          // and a verification email has not been sent by auth-server.
+          if (! account.get('verified') && isSignUp && ! emailSent) {
```

Comment:
That was my original plan, but decided against it for that reason, the js-client sets the default of `sendEmailIfUnverified=false` if it is not passed. Will update so that value gets propagated to client.